tests/src/ViewModel/ButtonSpeechBubbleTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace tests\eLife\Patterns\ViewModel;
+
+use eLife\Patterns\ViewModel\Button;
+
+final class ButtonSpeechBubbleTest extends ViewModelTest
+{
+    /**
+     * @test
+     */
+    public function it_has_data()
+    {
+        $data = [
+            'text' => '<span aria-hidden="true">3</span><span class="visuallyhidden">Open annotations (there are currently 3 annotations on this page).</span>',
```

Comment:
Looking like it's be useful to split this into two properties, or would `<span class="visuallyhidden">Open annotations (there are currently</span> 3 <span class="visuallyhidden">annotations on this page).</span>` work?

Reply:
Ah no, needs the symbol instead of 0.

Reply:
Could add a new function like `mixed_visibility_text()` instead? Not sure what to call it.

Reply:
See `mixed_accessibility_text()` https://github.com/elifesciences/patterns-php/pull/524/files/8a41c5346813bb7d5e4585a2fac9dabe441ff09c#diff-84b928a9905378810b602c8a740b4cbaR58

Keeping explicit expected text in test for documentation.